api/templates/teledeclaration_pdf.html

```diff
@@ -0,0 +1,225 @@
+                src:  url({% static '../../web/static/fonts/Marianne-Bold.ttf' %}) format('truetype');
+            }
+            @font-face {
+                font-family: 'Marianne-ExtraBold';
+                src: url({% static '../../web/static/fonts/Marianne-ExtraBold.ttf' %}) format('truetype');
+            }
+            body {
+                font-family: 'Marianne', Arial, Helvetica, sans-serif;
+            }
+            h1 {
+                font-family: 'Marianne-ExtraBold';
+                font-size: 24px;
+                margin: 32px 0px 16px 0px;
+            }
+            h2 {
+                font-family: 'Marianne-Bold';
+                font-size: 16px;
+                margin: 16px 0px;
+            }
+            h3 {
+                font-family: 'Marianne-Bold';
+            }
+            li span {
+                font-family: 'Marianne-Bold';
+            }
+            p, li {
+                margin: 16px 0px;
+            }
+            .image-container {
+                background-color:#0c7f46;
+                padding: 16px;
+            }
+            .other-measures > div {
+                margin-bottom: 10px;
+            }
+            .other-measures > div > span {
+                margin-right: 4px;
+            }
+            hr {
+                margin-top: 24px;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="image-container">
+            <img src="{% static '../../web/static/images/logo_transparent_white.png' %}" alt="ma cantine" height="30" width="142" />
+        </div>
+        <h1>Justificatif pour télédéclaration {{ year }}</h1>
+        <h2>Cantine {{ canteen_name }} (SIRET {{ siret }})</h2>
+        <h3>Télédéclaration faite le {{ date|date:"l m F Y" }} par {{ applicant }}</h3>
+
+        <p>
+            Conformément à l’article 24 de la loi EGAlim, chaque établissement est tenu de
+            renseigner et transmettre à l’administration ses données, notamment en termes
+            d’approvisionnement sur l’année civile passée.
+        </p>
+        <p>
+            Pour l'année {{ year }}, les données suivantes ont été télédéclarées le
+            {{ date|date:"l m F Y" }} via la plateforme numérique ma cantine :
+        </p>
+        <ul>
+            <li>
+                <span>Valeur totale en HT des achats alimentaires</span> : {{ value_total_ht|floatformat:2 }} €
+            </li>
+            <li>
+                <span>Valeur en HT des achats alimentaires en Bio</span> : {{ value_bio_ht|floatformat:2 }} €
+            </li>
+            <li>
+                <span>Valeur en HT d'autres produits de qualité et durables</span> : {{ value_sustainable_ht|floatformat:2 }} €
+            </li>
+            <li>
+                <span>Valeur en HT des produits issus du commerce équitable</span> : {{ value_fair_trade_ht|floatformat:2 }} €
+            </li>
+        </ul>
+
+        <div class="other-measures">
+            <div>
```

Comment:
Est-ce qu'on peut utiliser `for` pour tous ces points ? e.g.
```
{% for action in diagnostic_actions %}
<div>
{% if done %}
...
{% else %}
...
{% endif %}
{{ label }}
```

Reply:
Vu que on doit spécifier le texte "lisible" (par ex _"Plan d'action contre le gaspillage"_ qui correspond à `has_waste_plan`, ça se complique un peu. On peut créer le dictionnaire coté `view` et itérer dans le template (ce qui en vrai déplace le problème à un autre endroit), mais vu que je ne suis absolument pas sur du layout final qu'aura le justificatif, je voulais garder la view plus légère quite à re-faire des morceau du template dans un second temps.